File: Python Code/Daily_Flash_Code_0/Week 3/28Jan/13-DailyFlash_Solutions/13-DailyFlash_Solutions/28_Jan_Solutions_One/Cpp/pattern.cpp
#include<iostream>


int main(){

	int num =1;
	for(int outeritr = 0 ; outeritr < 4 ; outeritr++){
		
		for(int inneritr = 0 ; inneritr <= outeritr ; inneritr++ ){
			
			printf("%d\t",num*num*num);
			num++;
		}	
		printf("\n");
	}	
}	

File: Python Code/Daily_Flash_Code_0/Week 3/28Jan/13-DailyFlash_Solutions/13-DailyFlash_Solutions/28_Jan_Solutions_One/Java/Pattern.java
class Pattern{

	
	public static void main(String args[]){
		
		int num =1;
		for(int outeritr = 0 ; outeritr < 4 ; outeritr++){

			for(int inneritr = 0 ; inneritr <= outeritr ; inneritr++ ){

				System.out.printf("%d\t",num*num*num);
				num++;
			}
			System.out.printf("\n");
	
		}
	}	
}	
